src/arkode/arkode_ls.c

```diff
@@ -3186,19 +3250,16 @@ int arkLsMassSolve(void *arkode_mem, N_Vector b, realtype nlscoef)
     return(-1);
     break;
   case SUNLS_PACKAGE_FAIL_UNREC:
-    arkProcessError(ark_mem, SUNLS_PACKAGE_FAIL_UNREC, "ARKLS",
-                    "arkLsMassSolve",
+    arkProcessError(ark_mem, SUNLS_PACKAGE_FAIL_UNREC, __LINE__, __func__, __FILE__,
                     "Failure in SUNLinSol external package");
     return(-1);
     break;
   case SUNLS_ATIMES_FAIL_UNREC:
-    arkProcessError(ark_mem, SUNLS_ATIMES_FAIL_UNREC, "ARKLS",
-                    "arkLsMassSolve", MSG_LS_MTIMES_FAILED);
+    arkProcessError(ark_mem, SUNLS_ATIMES_FAIL_UNREC, __LINE__, __func__, __FILE__, MSG_LS_MTIMES_FAILED);
     return(-1);
     break;
   case SUNLS_PSOLVE_FAIL_UNREC:
-    arkProcessError(ark_mem, SUNLS_PSOLVE_FAIL_UNREC, "ARKLS",
-                    "arkLsMassSolve", MSG_LS_PSOLVE_FAILED);
+    arkProcessError(ark_mem, SUNLS_PSOLVE_FAIL_UNREC, __LINE__, __func__, __FILE__, MSG_LS_PSOLVE_FAILED);
     return(-1);
```

Comment:
As above, should each of these return values be converted to appropriate `SUNLsStatus` constants?